tutorials/notebooks/pytorch/gptq/example_pytorch_mobilenetv2_cifar100_gptq.ipynb

```diff
@@ -0,0 +1,717 @@
+        "trainloader = get_cifar100_trainloader(retrain_args.representative_dataset_dir,\n",
+        "                                       transform,\n",
+        "                                       retrain_args.retrain_batch_size)\n",
+        "\n",
+        "num_calibration_iterations = 10\n",
+        "def representative_data_gen() -> list:\n",
+        "    for _ in range(num_calibration_iterations):\n",
+        "        yield [next(iter(trainloader))[0]]"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "d0a92bee",
+      "metadata": {
+        "id": "d0a92bee"
+      },
+      "source": [
+        "In addition, MCT optimizes the model for dedicated hardware. This is done using TPC (for more details, please visit our [documentation](https://sony.github.io/model_optimization/docs/api/experimental_api_docs/modules/target_platform.html)). Here, we use the default Pytorch TPC:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "63f695dd",
+      "metadata": {
+        "id": "63f695dd"
+      },
+      "outputs": [],
+      "source": [
+        "# Get a TargetPlatformCapabilities object that models the hardware for the quantized model inference.\n",
+        "# Here, for example, we use the default platform that is attached to a Pytorch layers representation.\n",
+        "target_platform_cap = mct.get_target_platform_capabilities('pytorch', 'default')"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "d3521637",
+      "metadata": {
+        "id": "d3521637"
+      },
+      "source": [
+        "In order to use mixed-precision quantization we need to set some parameters in the CoreConfig that MCT uses:\n",
+        "1. Number of images - MCT uses images from the representative dataset to search for a suitable bit-width configuration. This parameter determine the number of images MCT will use. The more images, the bit-width configuration is expected to be more accurate (however this affects the search time, so there is a trade-off between runtime and expected accuracy).\n",
+        "2. Gradient weighting - A method to improve the bit-width configuration search (in exchange for longer search time). In this example, we will not use it."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "3ce7789b-aa3d-4a44-8dc5-dc052ece9cad",
+      "metadata": {
+        "id": "3ce7789b-aa3d-4a44-8dc5-dc052ece9cad"
+      },
+      "outputs": [],
+      "source": [
+        "# Create a mixed-precision quantization configuration with possible mixed-precision search options.\n",
+        "# MCT will search a mixed-precision configuration (namely, bit-width for each layer)\n",
+        "# and quantize the model according to this configuration.\n",
+        "# The candidates bit-width for quantization should be defined in the target platform model:\n",
```

Comment:
I would make the comments shorter (even remove it) since you already comment in the description above.